### PR TITLE
Apache Spark 2.3.0 Release: Announcements & Release Notes

### PR DESCRIPTION
**Pull request by sameeragarwal**
This patch adds the release notes and relevant announcements for Spark 2.3.0.

Release notes: https://github.com/sameeragarwal/spark-website/blob/100f9cbc90241c8fb618d83141396748f81868a5/releases/_posts/2018-02-28-spark-release-2-3-0.md

### CODE REVIEW

File: releases/_posts/2018-02-28-spark-release-2-3-0.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+   - **Vectorized ORC Reader**: [SPARK-16060] Adds support for new ORC reader that substantially improves the ORC scan throughput through vectorization (2-5x). To enable the reader, users can set `spark.sql.orc.impl` to `native`.
+   - **Spark History Server V2**: [SPARK-18085] A new spark history server (SHS) backend that provides better scalability for large scale applications with a more efficient event storage mechanism.
+   - **Data source API V2**: [SPARK-15689][SPARK-22386] An experimental API for plugging in new data sources in Spark. The new API attempts to address several limitations of the V1 API and aims to facilitate development of high performant, easy-to-maintain, and extensible external data sources. Note that this API is still undergoing active development and breaking changes should be expected.
+   - **PySpark Performance Enhancement**s: [SPARK-22216][SPARK-21187] Significant improvements in python performance and interoperability by fast data serialization and vectorized execution.
```
Comment (gatorsmile) [inline releases/_posts/2018-02-28-spark-release-2-3-0.md]: move `**` after `s`?

Comment (sameeragarwal) [reply releases/_posts/2018-02-28-spark-release-2-3-0.md]: good eye!

File: releases/_posts/2018-02-28-spark-release-2-3-0.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
...ebedev, Sergey Serebryakov, Sergey Zhemzhitsky, Seth Hendrickson, Shane Jarvie, Shashwat Anand, Shintaro Murakami, Shivaram Venkataraman, Shixiong Zhu, Shuangshuang Wang, Sid Murching, Sital Kedia, Soonmok Kwon, Srinivasa Reddy Vundela, Stavros Kontopoulos, Steve Loughran, Steven Rand, Sujith, Sujith Jay Nair, Sumedh Wale, Sunitha Kambhampati, Suresh Thalamati, Susan X. Huynh, Takeshi Yamamuro, Takuya UESHIN, Tathagata Das, Tejas Patil, Teng Peng, Thomas Graves, Tim Van Wassenhove, Travis Hegner, Tristan Stevens, Tucker Beck, Valeriy Avanesov, Vinitha Gankidi, Vinod KC, Wang Gengliang, Wayne Zhang, Weichen Xu, Wenchen Fan, Wieland Hoffmann, Wil Selwood, Wing Yew Poon, Xiang Gao, Xianjin YE, Xianyang Liu, Xiao Li, Xiaochen Ouyang, Xiaofeng Lin, Xiaokai Zhao, Xiayun Sun, Xin Lu, Xin Ren, Xingbo Jiang, Yan Facai (Yan Fa Cai), Yanbo Liang, Yash Sharma, Yinan Li, Yong Tang, Youngbin Kim, Yuanjian Li, Yucai Yu, Yuhai Cen, Yuhao Yang, Yuming Wang, Yuval Itzchakov, Zhan Zhang, Zhang A Peng, Zhaokun Liu, Zheng RuiFeng, Zhenhua Wang, Zuo Tingbing, brandonJY, caneGuy, cxzl25, djvulee, eatoncys, heary-cao, ho3rexqj, lizhaoch, maclockard, neoremind, peay, shaofei007, wangjiaochun, zenglinxi0615
```
Comment (gatorsmile) [inline releases/_posts/2018-02-28-spark-release-2-3-0.md]: > `Li, YanKit | Wilson | RIT,` ? 

What is this?

Comment (sameeragarwal) [reply releases/_posts/2018-02-28-spark-release-2-3-0.md]: auto-generated from this commit I believe: https://github.com/apache/spark/commit/6447d7bc1de4ab1d99a8dfcd3fea07f5a2da363d. Let me change it to Yan Kit Li (from https://www.linkedin.com/in/windkithk/)

**Comment by sameeragarwal** [comment]
Here's the updated page (with hyperlinks and some polish): https://github.com/sameeragarwal/spark-website/blob/100f9cbc90241c8fb618d83141396748f81868a5/releases/_posts/2018-02-28-spark-release-2-3-0.md

**Comment by gatorsmile** [comment]
LGTM

**Comment by sameeragarwal** [comment]
merged via https://github.com/apache/spark-website/commit/d1ea0db8ef6055ddbc24df687b1c1c2a01e002df

**Pull request closed without merging**